### PR TITLE
Windows: Update icu-easyrpg to 67.1

### LINKED ISSUE #94: Windows buildscript fails when installing harfbuzz (automake1.15-1.15.1-1)
I found an error when I try to setup windows toolchain; vcpkg fails when installing harfbuzz due to an issue when it tries to install automake.

Here's the log:

```
Installing package harfbuzz[core]:x64-windows-static... done
Elapsed time for package harfbuzz:x64-windows-static: 2.407 min
Starting package 9/25: icu-easyrpg:x64-windows-static
Building package icu-easyrpg[core]:x64-windows-static...
-- Using cached C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/downloads/icu4c-65_1-src.tgz
-- Cleaning sources at C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/buildtrees/icu-easyrpg/src/c-65_1-src-6a069efe25.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/downloads/icu4c-65_1-src.tgz
-- Applying patch C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/ports/icu-easyrpg/disable-escapestr-tool.patch
-- Applying patch C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/ports/icu-easyrpg/remove-MD-from-configure.patch
-- Applying patch C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/ports/icu-easyrpg/fix_parallel_build_on_windows.patch
-- Applying patch C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/ports/icu-easyrpg/fix-extra.patch
-- Using source at C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/buildtrees/icu-easyrpg/src/c-65_1-src-6a069efe25.clean
-- Using cached C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/downloads/icudata.tar.gz
-- Extracting source C:/Users/user/Desktop/easyrpg/buildscripts/windows/vcpkg/downloads/icudata.tar.gz
-- Downloading https://www.python.org/ftp/python/3.9.0/python-3.9.0-embed-amd64.zip...
-- Downloading https://repo.msys2.org/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.ustc.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://mirrors.ustc.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.bit.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://mirror.bit.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.selfnet.de/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://mirror.selfnet.de/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:184 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:90 (vcpkg_download_distfile)
  scripts/cmake/vcpkg_acquire_msys.cmake:128 (msys_package_download)
  scripts/cmake/vcpkg_acquire_msys.cmake:278 (msys_package)
  ports/icu-easyrpg/portfile.cmake:92 (vcpkg_acquire_msys)
```

Effecitvely, all these mirrors throws 404 errors. Taking a peek, these pages now offers automake1.15-1.15.1-2 (but automake1.15-1.15.1-1 seems away). Maybe a non-updated database or some sort of hot unexpected release?

### PR DESCRIPTION
**Pull request by carstene1ns**
This is untested! Brings our custom icu on par with vcpkg upstream.
(The easyrpg specific part - custom data file and static data packaging - is retained)

Fixes #94.
The actual fix is updating to automake 1.16, if needed, we could apply this without updating the rest as a quick fix.
See also: https://github.com/microsoft/vcpkg/pull/13980/files

### CODE REVIEW

File: windows/icu-easyrpg/portfile.cmake [hunk 1]
```diff
@@ -234,9 +287,7 @@ endif()
...
 # This is expected because ICU database contains no executable code
 vcpkg_copy_pdbs()
+vcpkg_fixup_pkgconfig(SYSTEM_LIBRARIES pthread m)
 
 # Handle copyright
-# EASYRPG CUSTOM
-file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/icu-easyrpg)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/icu-easyrpg/LICENSE ${CURRENT_PACKAGES_DIR}/share/icu-easyrpg/copyright)
-# EASYRPG END
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (carstene1ns) [inline windows/icu-easyrpg/portfile.cmake:293]: Not sure why here a customization was needed, it should be the same with this upstream line.

Comment (Ghabry) [reply windows/icu-easyrpg/portfile.cmake:293]: There is a sanity check that makes the build fail when the license file is missing. The $PORT hopefully contains the correct content

**Comment by carstene1ns** [comment]
Split out the automake update now, since it unbreaks the toolchain build.
The actual update here can stay longer as it needs testing.

**Comment by Ghabry** [comment]
Sorry, forgot about this PR.
I do now a test build and will report back if the Player works, thanks.

**Comment by Ghabry** [comment]
@carstene1ns 

I'm currently unable to test it because CMake refuses now to find 32-bit fmt o_O. I try to find the problem.

There is another build failure: Could you also add the following lines to the build.cmd please? On the server I added a hack for this months ago because yasm was not found but looks like there is now an upstream fix for this:

```
CMake Error at installed/x64-windows-static/share/yasm-tool-helper/yasm-tool-helper.cmake:16 (message):
  Cross-targetting and x64 ports requiring yasm require the x86-windows
  yasm-tool to be available.  Please install yasm-tool:x86-windows first.
Call Stack (most recent call first):
  ports/mpg123/portfile.cmake:37 (yasm_tool_helper)
  scripts/ports.cmake:136 (include)
```

```
diff --git a/windows/build.cmd b/windows/build.cmd
index 1103e99..702f9a1 100644
--- a/windows/build.cmd
+++ b/windows/build.cmd
@@ -2,6 +2,10 @@

 call helper\prepare.cmd

+:: Install yasm (required for 64 bit mpg123 build)
+vcpkg install --triplet x86-windows^
+ yasm-tool[core]^M
+
 :: Build 32-bit libraries
 :: Using [core] everywhere to prevent surprises when new default-features are
 :: added to libraries.
```

**Comment by Ghabry** [comment]
The fmt error was just me not reading. CMake requires now for Visual Studio solutions that the architecture is specified via "-A" argument.

**Review by Ghabry — APPROVED** [review]
works for me. Thanks!

**Pull request merged**